doc/source/how-to/setting-up.rst

```diff
@@ -467,6 +467,41 @@ For information on setting up ``SSH`` with ``GitHub``, in the ``GitHub`` documen
 see `Connecting to GitHub with SSH
 <https://docs.github.com/en/authentication/connecting-to-github-with-ssh>`_.
 
+Handling line endings
+~~~~~~~~~~~~~~~~~~~~~
+Every time you introduce a new line by pressing the `return` key, an invisible
+character is introduced to represent a line ending. Each operating system manages
+these end of line (EOL) characters in their own way. For Windows, the EOL is
+also known as ``CRLF`` while in Linux its name is ``LF``.
```

Comment:
```suggestion
also known as a ``CRLF``, while in Linux its known as a ``LF``.
```

Reply:
Here you've used double back ticks but since we are referring to concepts (rather than some code), I'd recommend using one back tick or one asterisk to italicize. That's what I've been dong for new terms and concepts.

Reply:
Thanks for the suggestion, @PipKat. I'm applying it.